docs/content/users/debugging-profiling/step-debugging.md

```diff
@@ -118,6 +118,7 @@ Here are basic steps to take to sort out any difficulty:
 * Reboot your computer.
 * If you're running WSL2 and have PhpStorm running inside WSL2 (the Linux version of PhpStorm) then `ddev config global --xdebug-ide-location=wsl2`.
 * Temporarily disable any *firewall* or *VPN* if you’re having trouble. Xdebug is a network protocol, and the PHP process inside the web container must be able to establish a TCP connection to the listening IDE (PhpStorm, for example).
+* Confirm that DDEV’s [`xdebug_ide_location`](../../users/configuration/config/#xdebug_ide_location) config setting is set properly, which in most cases should be set to an empty string. Check both your project's `.ddev/config.yaml` and DDEV’s global `~/.ddev/global_config.yaml`.
```

Comment:
Pretty sure this should be: (anyway, failed link check)

```suggestion
* Confirm that DDEV’s [`xdebug_ide_location`](../../configuration/config/#xdebug_ide_location) config setting is set properly, which in most cases should be set to an empty string. Check both your project's `.ddev/config.yaml` and DDEV’s global `~/.ddev/global_config.yaml`.
```

Reply:
Wow, I checked it locally so it’s kind of amazing I still managed to get it wrong. Thanks for fixing!

Reply:
It took me three tries. Then go back to the IDE locally, where it helped me get the right syntax. It would help if the markdown syntax were the same as http syntax.